src/components/Navbar/Navbar.js

```diff
@@ -1,16 +1,54 @@
 import React from "react";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+
+import { ShoppingCart, ChevronDown } from "react-feather";
+
+import Searchbar from "components/SearchBar/Searchbar";
+
+import logo from "assets/images/logo.png";
 
 import styles from "./Navbar.module.scss";
 
-function Navbar() {
+function Navbar(props) {
   return (
     <div className={styles.navbar}>
-      <div className={styles.logo}></div>
-      <div className={styles.categories}></div>
-
-      <div className={styles.authSection}></div>
+      <div className={styles.logo}>
+        <img src={logo} alt="logo" />
+      </div>
+      <div className={styles.content}>
+        <div className={styles.categories}>
+          <span>
+            Categories
+            <ChevronDown />
+          </span>
+        </div>
+        <Searchbar />
+        <ShoppingCart />
+        <div className={styles.authSection}>
+          {!props.auth ? (
+            <>
+              <Link to="/merchant/register">Signup</Link>
+              <span>/</span>
+              <Link to="/merchant/login">Login</Link>
```

Comment:
why these link points to only merchant signup login ?

dont change it now but just we need to discuss on it